external/Core/CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ add_custom_target(CoreExternals ALL)
 
 if(NOT DEFINED Eigen3_DIR)
     check_externals_prerequisite()
-    status_message("" "eigen3" "remote git")
+    status_message("[CoreExternal]" "eigen3" "remote git")
```

Comment:
I have made a proposal to have better external message in #1016, may superseed this.